src/Umbraco.Web.UI.NetCore/Controllers/TestBackOfficeUmbracoApiController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Web.Common.Attributes;
+using Umbraco.Web.Common.Controllers;
+
+namespace Umbraco.Web.UI.NetCore.Controllers
+{
+    [PluginController("Test")]
+    [IsBackOffice]
+    public class TestBackOfficeUmbracoApiController : UmbracoApiController
```

Comment:
I guess this not should have been committed

Reply:
I left it there just for testing but will remove, if you want to test you can add your own.